### DISCUSSION TITLE
Sales comparison by month

### DISCUSSION THREAD

**Discussion by krseguros** [YetiForceCompany/YetiForceCRM#17490] [How to]
Hello,
I need to know from which module the results of this Widgets come out?
https://gitstable.yetiforce.com/index.php
![image](https://github.com/YetiForceCompany/YetiForceCRM/assets/96985476/376800e6-b14e-4c09-ab0c-fe100a3a2135)

**Comment by rskrzypczak** [comment]
Invoice
